events/nats.go

```diff
@@ -218,6 +214,17 @@ func (n *NatsJetstream) addConsumer() error {
 		FilterSubject: n.parameters.Consumer.FilterSubject,
 	}
 
+	// Update consumer configuration when one exists
+	for name := range n.jsctx.ConsumerNames(n.parameters.Stream.Name) {
```

Comment:
Are we opening ourselves up for a hassle when multiple controllers are restarted and attempt to update the consumer concurrently?

Reply:
with the latest change, it'll be a race between the workers to update the configuration so it should be ok ?

Reply:
Probably, yes.